chapters/arrays.tex

```diff
@@ -216,20 +216,20 @@ \section{Built-in Array Functions}\doublelabel{built-in-array-functions}
 \begin{longtable}[]{|l|p{9cm}|}
 \caption{Promote function (cannot be used in Modelica).}\\
 \hline \endhead
-promote(A,n) & Fills dimensions of size 1 from the right to array A upto
-dimension n, where "n \textgreater{}= ndims(A)" is required. Let C =
-promote(A,n), with nA=ndims(A), then ndims(C) = n, size(C,j) = size(A,j)
-for 1 \textless{}= j \textless{}= nA, size(C,j) = 1 for nA+1 <= j <= n, C{[}i\_1, ..., i\_nA, 1, ..., 1{]} =A{[}i\_1, ..., i\_nA{]}\\ \hline
+\lstinline!promote(A, n)! & Fills dimensions of size 1 from the right to array \lstinline!A! upto
+dimension $n$, where $n \geq$ \lstinline!ndims(A)! is required. Let \lstinline!C = promote(A, n)!, with \lstinline!nA = ndims(A)!, then \lstinline!ndims(C) = n!, \lstinline!size(C, j) = size(A, j)! for $1 \leq \text{\lstinline!j!} \leq \text{\lstinline!nA!}$, $\text{\lstinline!size(C, j)!} = 1$ for $\text{\lstinline!nA!} + 1 \leq \text{\lstinline!j!} \leq \text{\lstinline!n!}$, \lstinline!C[i_1, ..., i_nA, 1, ..., 1]! = \lstinline!A[i_1, ..., i_nA]!\\ \hline
```

Comment:
I get the general idea of using code and math like this, but it seems problematic in the small table for LaTeXML

Reply:
The small table is a problem in general, but I can't see how this use of code and math would make it worse.  Regardless of this, it could be that the small table size is actually forcing us to introduce line breaks for improved readability… However, I'd prefer finding a better layout in general than start introducing line breaks that are just going to cause problems the day we start using a better layout.